descriptors/ModuleDescriptor-template.json

```diff
@@ -26,9 +26,130 @@
           "pathPattern": "/_/tenant"
         }
       ]
+    },
+    {
+      "id": "patron-block-conditions",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/patron-block-conditions",
+          "permissionsRequired": ["patron-block-conditions.collection.get"]
+        }, {
+          "methods": ["GET"],
+          "pathPattern": "patron-block-conditions/{id}",
+          "permissionsRequired": ["patron-block-conditions.item.get"]
+        }, {
+          "methods": ["POST"],
+          "pathPattern": "/patron-block-conditions",
+          "permissionsRequired": ["patron-block-conditions.item.post"]
+        }, {
+          "methods": ["PUT"],
+          "pathPattern": "/patron-block-conditions/{id}",
+          "permissionsRequired": ["patron-block-conditions.item.put"]
+        }, {
+          "methods": ["DELETE"],
+          "pathPattern": "/patron-block-conditions/{id}",
+          "permissionsRequired": ["patron-block-conditions.item.delete"]
+        }
```

Comment:
I think we should remove these endpoints. As Zak suggested several times, it doesn't make sense to add new conditions and removing them can be dangerous.

Reply:
Updated, thanks